src/components/widget/SelectionDropdown.js

```diff
@@ -14,19 +15,7 @@ export default class SelectionDropdown extends Component {
         size: PropTypes.number.isRequired,
       }),
     ]).isRequired,
-    selected: (props, propName) => {
```

Comment:
@PabloSichert can you throw some light on this part ? It looks like a huge overcomplication for something that should be pretty straightforward. As far as I know we expect `selected` to be an object, but I might be missing something. This on the other hand looks like it wants to handle different cases and work for different props (thus the propName I suppose) but will only work for `selected` prop and from my testing it always gets an object or `null`.

Reply:
It checks if `selected` is either one of `options` or `null/undefined`. So where you got the errors, something different was passed in, which should prevent the component from working properly.

No, it's only supposed to work for "selected" - you are right in that it should rather have been hardcoded there